libs/sdk-ui-dashboard/src/model/events/drillTargets.ts

```diff
@@ -0,0 +1,45 @@
+// (C) 2021 GoodData Corporation
+import { ObjRef } from "@gooddata/sdk-model";
+import { IAvailableDrillTargets } from "@gooddata/sdk-ui";
+import { DashboardContext } from "../types/commonTypes";
+import { IDashboardEvent } from "./base";
+
+/**
+ *  Widget drill targets added event
+ *
+ * @internal
```

Comment:
All the `@internal` should be `@alpha` in this file after #1562

Reply:
Done